packages/fxa-content-server/app/scripts/lib/experiments/grouping-rules/signup-code.js

```diff
@@ -27,6 +27,11 @@ const ROLLOUT_CLIENTS = {
     name: 'Lockbox Extension iOS',
     rolloutRate: 0.0,
   },
+  e6eb0d1e856335fc: {
```

Comment:
This is the client_id of the VPN, do you also want this feature for the proxy? The proxies new client_id will be 565585c1745a144d.

Reply:
Landing this as is but @johngruen thoughts?